src/System.Text.Json/src/System/Text/Json/Node/JsonNodeOptions.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace System.Text.Json
+{
+    /// <summary>
+    /// Provides the ability for the user to define custom behavior when parsing JSON to create a <see cref="JsonNode"/>.
+    /// </summary>
+    public struct JsonNodeOptions
+    {
+        internal const int DefaultMaxDepth = 64;
+
+        private int _maxDepth;
+        private JsonCommentHandling _commentHandling;
+
+        /// <summary>
+        /// Defines how the <see cref="Utf8JsonReader"/> should handle comments when reading through the JSON.
```

Comment:
We should change this to match the actual JsonDocument documentation (i.e. don't use `Utf8JsonReader` here):
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsondocumentoptions.commenthandling?view=netcore-3.0#System_Text_Json_JsonDocumentOptions_CommentHandling

Reply:
Hmm, I got this documentation from `JsonDocumentOptions` class summary (Just copying the whole file and adding a new option), but I can change it.

Reply:
The content from the dotnet-api-docs repo is the source of truth (and has gone through review/edits from the docs team). If you want to start off with a baseline, it's fine to use the `JsonDocumentOptions` documentation, but do so from the docs we publish (and not the /// comments within the source, which are out-dated).